doc/user-guide/io.rst

```diff
@@ -279,6 +279,16 @@ You can view this encoding information (among others) in the
 Note that all operations that manipulate variables other than indexing
 will remove encoding information.
 
+In some cases it is useful to intentionally reset a dataset's original encoding values.
+This can be done with either the :py:meth:`Dataset.reset_encoding` or
+:py:meth:`DataArray.reset_encoding` methods.
+
+.. ipython::
+    :verbatim:
+
+    In [1]: ds_no_encoding = ds_disk.reset_encoding()
+    In [2]: ds_no_encoding["y"]
+    Out[2]: {}
```

Comment:
I wonder if we can somehow create a working example of this? There's already a bug here that would have been caught by executing: it should be `ds_no_encoding["y"].encoding`. Maybe something like
```python
ds = xr.Dataset({"y": ("x", [1.1, 1.2], {}, {"add_offset": 1, "scale_factor": 0.1, "dtype": "int8"})})
ds_no_encoding = ds.reset_encoding()
ds_no_encoding["y"].encoding
```
?